packages/core/src/app/checkout/Checkout.tsx

```diff
@@ -180,6 +180,12 @@ class Checkout extends Component<
         this.handleBeforeExit();
     }
 
+    componentDidUpdate(prevProps: WithCheckoutProps): void {
+        if(prevProps.steps.length === 0 && this.props.steps && this.props.steps.length > 0) {
+            this.handleReady();
+        }
+    }
```

Comment:
This change is necessary. Otherwise, the checkout cannot go to the customer step because `steps` is `[]` when `this.handleReady();` is invoked.